index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="assets/css/reset.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>CLP | Psychic Game</title>
+</head>
+
+<body>
+
+    <div id="game">
+        <h1>Guess what letter I'm thinking of</h1>
+        <h2>Press any key to start!</h2>
+        <hr>
+        <p id="wins-text"></p>
```

Comment:
You can always add in your Wins: and etc here and just append or change other parts of the text here